maat-court-data-api/src/main/java/gov/uk/courtdata/job/QueueMessageMaintenanceScheduler.java

```diff
@@ -29,13 +31,15 @@ public class QueueMessageMaintenanceScheduler {
     @Scheduled(cron = "${queue.message.log.cron.expression}")
     @Transactional
     public void purgeHistory() {
-
+        String laaTransactionId = UUID.randomUUID().toString();
+        LoggingData.LAA_TRANSACTION_ID.putInMDC(laaTransactionId);
         log.info("Start Queue Message Purge Job...");
 
         LocalDateTime currentDate = LocalDateTime.now();
 
         queueMessageLogRepository.deleteCreatedOnOrBefore(currentDate.minusDays(getExpiryInDays()));
 
         log.info("Finish Queue Message Purge Job...");
+        LoggingData.LAA_TRANSACTION_ID.putInMDC("");
```

Comment:
I'm not sure this is purging the MDC if that's what you're trying to do? Wouldn't it be removeFromMdc by the laaTransactionId you created?

Reply:
Good spot. That was the old way I was doing it until I created the removeFromMdc method - just forgot to update. Updated now :)